modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -22,7 +22,7 @@ By default, {product-title} supports CSI volume cloning with these limitations:
 * The source and destination storage class must be the same.
 * Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
 * {product-title} does not include any CSI drivers. Use the CSI drivers provided by
-link:https://kubernetes-csi.github.io/docs/drivers.html[community or storage vendors]. Follow the installation instructions provided by the CSI driver.
+link:https://kubernetes-csi.github.io/docs/drivers.html[community or storage vendors]. Follow the installation instructions provided by the CSI driver provider.
```

Comment:
I thought we were supposed to be avoiding links to github in the docs?

Reply:
I'm guessing that this URL was previously admitted into our docs because it is for documentation published on github.io, not a repo on github.com. Here, we're simply improving the phrase. I've updated it to "Follow the installation instructions furnished by the CSI driver provider."